.gitmodules

```diff
@@ -1,7 +1,3 @@
 [submodule "open-simulation-interface"]
 	path = open-simulation-interface
 	url = https://github.com/OpenSimulationInterface/open-simulation-interface.git
-
-[submodule "osi-sensor-model-packaging"]
-	path = osi-sensor-model-packaging
-	url = https://github.com/OpenSimulationInterface/osi-sensor-model-packaging.git
```

Comment:
Was this never used?

Reply:
Not sure I mean we don't have it as a subfolder now and for installation it is not needed.